scripts/run_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [[ -z "$1" ]]; then
+  echo "USAGE: run_tests.sh PLATFORM DEVICE OS [XCODE_VERSION]"
+  echo "EXAMPLE: run_tests.sh "iOS Simulator" "iPhone 14 Pro" 16.2 14.2"
+  exit 1
+fi
+
+PLATFORM=$1
+DEVICE=$2
+OS=$3
+
+if [[ ! -z "$4" ]]; then
+  XCODE_PATH="/Applications/Xcode_$4.app"
+  sudo xcode-select -switch $XCODE_PATH && /usr/bin/xcodebuild -version
+fi
+
+DESTINATION="platform=$PLATFORM"
+DESTINATION="$DESTINATION,name=$DEVICE"
+DESTINATION="$DESTINATION,OS=$OS"
+  
+# Run tests
+xcodebuild test -project xcode/Subconscious/Subconscious.xcodeproj \
+  -scheme "Subconscious (iOS)" \
```

Comment:
I tried to come up with a Scheme that _doesn't_ launch the simulator to run our tests. They run within Xcode during local dev without opening the simulator... but I had no luck. Maybe @gordonbrander knows more?

We could save a little CI time if we didn't run the `iOS` specific tests like `testLaunchPerformance` but I'm not sure that's worth it.

Reply:
I've disabled `testLaunchPerformance` which trimmed 2 minutes off the CI workflow.

Reply:
Fantastic!